pages/api-reference/funding-sources/index.mdx

```diff
@@ -17,11 +17,7 @@ The Funding Sources resource represents payment accounts that can be used to sen
 
 ### Funding source types
 
-The three funding source types available with a Dwolla integration include a `bank`, the Dwolla `balance`, and a debit `card`. Type `bank` represents any bank account attached as a funding source to Account and Customer resources. Type `balance` represents the Dwolla Balance made available to Account and Verified Customer resources. Type `card` represents a reloadable prepaid or bank-issued debit card that can be used as a funding source type to receive funds in near real-time.
-
-##### Balance funding source
-
-The [Dwolla Balance](https://developers.dwolla.com/resources/balance-funding-source.html) can be utilized as a digital "wallet", storing USD funds for the Customer or Account with Dwolla’s financial institution partners. Additionally, the Dwolla Balance can be pre-loaded with funds for quicker outgoing ACH transfers to destination funding sources. To get a more in-depth overview of the Dwolla Balance, including functionality and other benefits, check out our [developer resource article](https://developers.dwolla.com/resources/balance-funding-source.html) or view our [webinar](https://www.dwolla.com/resources/balance-webinar/).
+The four funding source types available with a Dwolla integration include a `bank`, the Dwolla `balance`, a `VAN`, and a debit `card`. Type `bank` represents any bank account attached as a funding source to Account and Customer resources. Type `balance` represents the Dwolla Balance made available to Account and Verified Customer resources. Type `virtual` represents a Virtual Account Number (VAN) funding source which can be used outside of the Dwolla Platform to route ACH payments both to and from a Dwolla Balance. Type `card` represents a reloadable prepaid or bank-issued debit card that can be used as a funding source type to receive funds in near real-time.
```

Comment:
Correction: The four funding source types available with a Dwolla integration include a `bank`, the Dwolla `balance`, a `virtual`, and a debit `card`.